godtools/Views/TractElements/TractCallToAction.swift

```diff
@@ -47,16 +47,15 @@ class TractCallToAction: BaseTractElement {
     }
     
     override func loadFrameProperties() {
-        var yPosition: CGFloat {
-            var position = self.elementFrame.y + TractCallToAction.yMarginConstant
-            if position < (self.parent?.getMaxHeight())! - self.height {
-                position = (self.parent?.getMaxHeight())! - self.height
-            }
-            return position
+        let bottomConstant = TractPageContainer.marginBottom
+        let minPosition = (self.parent?.getMaxHeight())! - self.height - bottomConstant
```

Comment:
We should be able to guarantee that the parent is defined so force-unwrapping is okay. I'd prefer to see it written this way though, as it's easier to read:
````swift
let minPosition = self.parent!.getMaxHeight() - self.height - bottomConstant